app/src/main/java/io/appium/settings/receivers/AbstractSettingReceiver.java

```diff
@@ -45,6 +45,11 @@ public void onReceive(Context context, Intent intent) {
             return;
         }
         boolean isSuccessful;
+        AbstractSettingHandler handler = getHandler(context);
+        if (handler == null) {
+            setResultCode(Activity.RESULT_CANCELED);
+            return;
+        }
         if (command.equals(COMMAND_ENABLE)) {
             isSuccessful = getHandler(context).enable();
```

Comment:
we can use `handler` here instead of calling getHandler again

Reply:
done